src/ember-app.js

```diff
@@ -136,8 +136,12 @@ class EmberApp {
       let isWhitelisted = whitelist.indexOf(packageName) > -1;
 
       if (isWhitelisted) {
-        let resolvedModulePath = resolve.sync(moduleName, { basedir: distPath });
-        return require(resolvedModulePath);
+        try {
+          let resolvedModulePath = resolve.sync(moduleName, { basedir: distPath });
+          return require(resolvedModulePath);
+        } catch (error) {
+          return require(moduleName);
```

Comment:
We should tweak this to something akin to:

```suggestion
          if (error.code === 'MODULE_NOT_FOUND') {
            return require(moduleName);
          } else {
            throw error;
          }
```